sky/packages/sky/lib/gestures/arena.dart

```diff
@@ -40,10 +42,13 @@ class GestureArena {
 
   void _resolve(Object key, GestureArenaMember member, GestureDisposition disposition) {
     List<GestureArenaMember> members = _arenas[key];
+    if (members == null && disposition == GestureDisposition.accepted)
+      return;
     assert(members != null);
     assert(members.contains(member));
```

Comment:
In the case of the mouse pointer, it will go down multiple times with the same pointer ID. One could imagine a case where an arena has been resolved for that pointer, but where one of the gestures is still alive (a fling, say), and then later the pointer goes down again, and coincidentally (e.g. on a timer) the old gesture tries to accept or reject based on the old information, and so you end up here with a member of an old obsolete arena but with the pointer ID of a newly recreated arena.